ext_emconf.php

```diff
@@ -8,14 +8,5 @@
     'author_email' => 'gr@studiomitte.com',
     'state' => 'beta',
     'clearCacheOnLoad' => true,
-    'version' => '0.2.0',
-    'constraints' =>
-        [
-            'depends' => [
-                'typo3' => '11.5.0-11.5.99',
```

Comment:
please leave the constraints but adopt:
- typo3: 12.4.0-12.4.99
- cart: 9.0.0-9.99.99